source/features/toggle-files-button.tsx

```diff
@@ -49,5 +57,6 @@ void features.add({
 	include: [
 		pageDetect.isRepoTree
 	],
+	awaitDomReady: false,
```

Comment:
```suggestion
	awaitDomReady: false,
	repeatOnBackButton: true,
```

This is needed if not the toggle does not work once the back is pressed

Reply:
Done!